README.md

```diff
@@ -1,21 +1,48 @@
+<!--
+:type: service
+:name: TICK Stack GCP
```

Comment:
My understanding is that the name should be just "TICK Stack", and the GCP nature of it is pulled in by the `cloud` tag. But I could be wrong: @eak12913 is that a correct assumption?

Reply:
@yorinasub17 - that's correct.

Reply:
@eak12913 is `:name: TICK Stack` still valid for the `README-InfluxDB.md`, or should it be `InfluxDB`?